aanouncing-python-sdk-ga.mdx

```diff
@@ -0,0 +1,60 @@
+
+## Speed improvements
+
+We changed a couple of things under the hood about how connections being managed and reused. The refactoring has yielded significant performance improvements across the board. The most significant ones are:
+
+| Operation | Speedup (on average)|
+| --- | --- |
+| Get a single record | 5.95x |
+| Insert a single record | 4.95x |
+| Insert 100 records with transactions | 2.22x |
+
+## PEP-8 FTW!
+
+We received feedback that the SDK didn't not feel pythonic, and we couldn’t agree more! Initially, in the `0.x` releases, the API was generated one by one from [our OpenAPI specification](https://xata.io/docs/rest-api/openapi), which resulted in a non-pythonic API. For this release, we have aligned as much as possible with the [PEP-8](https://peps.python.org/pep-0008/) standard.
+
+## How to migrate to 1.x?
+
+Migrating to a new major version can be painful to get up to speed on the breaking changes and dropped functionality. With that in mind, we were trying to minimize the impact while still making necessary changes for the good. You can check out the [full migration guide](https://xata.io/docs/python-sdk/migration-guide) in our docs.
+
+The most impactful user-facing change is the renaming of the API surface [[xata-py#93](https://github.com/xataio/xata-py/issues/93)] and considered a breaking change. Additionally, some API endpoint calls were simplified to remove unnecessary code bloat, like creating a database.
+
+Previously in `0.x`, you needed to investigate the payload shape and be specific about the region and branch name.
+
+```python
+xata.databases().createDatabase("new_db",
+  {
+    "region": "us-east-1",
+    "branchName": "main",
+  }
+)
+```
+
+In version `1.0.*`, we've redesigned the API interface to incorporate payload options directly into the method signature. We've also optimized the SDK's internal values to make assumptions to reuse the SDK internal values, like the `region`. This results in a more streamlined API structure, and you can achieve functionality using:
+
+```python
+xata.databases().create("new_db")
+```
+
+## What happens to 0.x?
+
+We are going to continue to support the `0.x` version of the Xata Python SDK with maintenance releases for one more year. New API and security fixes will be added but no helpers or other improvements will be backported. The `0.x` product line of the SDK will be sunsetted on September 1st, 2024.
+
+## What’s next?
+
+Check out our [documentation](https://xata.io/docs/python-sdk/migration-guide) and let us know what you think. We’d love to hear from you! If you think something is missing or found a bug, open a ticket in the [xata-py](https://github.com/xataio/xata-py) repository. All contributions are welcome. You can also follow us on [Twitter](https://twitter.com/xata) or join us in [Discord](https://xata.io/discord).
+
```

Comment:
There's an opportunity here to remind folks about our other SDKs, likely before these closing statements. Definitely TypeScript, but would could highlight the community Go and Rust SDKs as well.

https://github.com/kerdokurs/xata-go
https://github.com/quanturtle/xata-rs

I'm not sure what the current state of the rust SDK is, but we can [follow up in Discord](https://discord.com/channels/996791218879086662/1001924127705272361/1116809661404958820).

Reply:
The rust one hasn't moved much and only supports the `/user` endpoint. Let's leave that one out. I'll add the go SDK and the TF provider.